Parallel Make for RI5CY

The `riscv-target/ri5cy` doesn't work due to changes that made parallel Make work.

```bash
export RISCV_PREFIX=riscv32-unknown-elf-
export VSIM_WORK=/home/bluew/projects/riscv/tb/core/work/
export TARGET_SIM=/home/bluew/projects/riscv/tb/core/testbench_verilator
export RISCV_DEVICE=rv32imc
export RISCV_TARGET=ri5cy
make RISCV_ISA=rv32i && make RISCV_ISA=rv32im && make RISCV_ISA=rv32imc
```
results in
```bash
make  \
        RISCV_TARGET=ri5cy \
        RISCV_DEVICE=rv32imc \
        RISCV_PREFIX=riscv32-unknown-elf- \
        run -C /home/bluew/tmp/riscv-compliance/riscv-test-suite/rv32i
make[1]: Entering directory `/home/bluew/tmp/riscv-compliance/riscv-test-suite/rv32i'
Compile /home/bluew/tmp/riscv-compliance/work/rv32i/I-ENDIANESS-01.elf
/home/bluew/.riscv/lib/gcc/riscv32-unknown-elf/9.2.0/../../../../riscv32-unknown-elf/bin/ld: cannot open output file /home/bluew/tmp/riscv-compliance/work/rv32i//home/bluew/tmp/riscv-compliance/work/rv32i/I-ENDIANESS-01.elf: No such file or directory
collect2: error: ld returned 1 exit status
/bin/sh: /home/bluew/tmp/riscv-compliance/work/rv32i//home/bluew/tmp/riscv-compliance/work/rv32i/I-ENDIANESS-01.elf.objdump: No such file or directory
/bin/sh: /home/bluew/tmp/riscv-compliance/work/rv32i//home/bluew/tmp/riscv-compliance/work/rv32i/I-ENDIANESS-01.elf.readelf: No such file or directory
/bin/sh: /home/bluew/tmp/riscv-compliance/work/rv32i//home/bluew/tmp/riscv-compliance/work/rv32i/I-ENDIANESS-01.elf.nm: No such file or directory
riscv32-unknown-elf-objcopy: '/home/bluew/tmp/riscv-compliance/work/rv32i//home/bluew/tmp/riscv-compliance/work/rv32i/I-ENDIANESS-01.elf': No such file
make[1]: *** [/home/bluew/tmp/riscv-compliance/work/rv32i/I-ENDIANESS-01.elf] Error 1
make[1]: Leaving directory `/home/bluew/tmp/riscv-compliance/riscv-test-suite/rv32i'
make: *** [simulate] Error 2
```

The following commit introduced the problem/regression @eroom1966  :
```
fba4324e62a85bd5de5c73d58d1ebd8edfb00da9 is the first bad commit
commit fba4324e62a85bd5de5c73d58d1ebd8edfb00da9
Author: eroom1966 <moore@imperas.com>
Date:   Mon Feb 17 10:03:59 2020 +0000

    Add Parallel make for supported targets
```

--- a/riscv-target/ri5cy/device/rv32imc/Makefile.include
+++ b/riscv-target/ri5cy/device/rv32imc/Makefile.include
@@ -8,9 +8,9 @@ TARGET_FLAGS ?= $(RISCV_TARGET_FLAGS)
 
 RUN_TARGET=\
     $(TARGET_SIM) $(TARGET_FLAGS) -c -quiet tb_top_vopt -do 'run -all; exit -f' \
-        +firmware=$(work_dir_isa)/$<.bin \
+        +firmware=$<.bin \
 	+maxcycles=10000 \
-	+signature=$(work_dir_isa)/$(*).signature.output
+	+signature=$(*).signature.output
 
 RISCV_PREFIX   ?= riscv32-unknown-elf-
 RISCV_GCC      ?= $(RISCV_PREFIX)gcc
@@ -21,13 +21,13 @@ RISCV_READELF  ?= $(RISCV_PREFIX)readelf
 RISCV_GCC_OPTS ?= -static -mcmodel=medany -fvisibility=hidden -nostdlib -nostartfiles $(RVTEST_DEFINES)
 
 COMPILE_TARGET=\
-	$$(RISCV_GCC) $(2) $$(RISCV_GCC_OPTS) \
+	$$(RISCV_GCC) $(1) $$(RISCV_GCC_OPTS) \
 		-I$(ROOTDIR)/riscv-test-env/ \
 		-I$(ROOTDIR)/riscv-test-env/p/ \
 		-I$(TARGETDIR)/$(RISCV_TARGET)/ \
-		$(DEFINES) -T$(LDSCRIPT) $$< \
-		-o $(work_dir_isa)/$$@; \
-    $$(RISCV_OBJDUMP) -D $(work_dir_isa)/$$@ > $(work_dir_isa)/$$@.objdump; \
-    $$(RISCV_READELF) -a $(work_dir_isa)/$$@ > $(work_dir_isa)/$$@.readelf; \
-    $$(RISCV_NM) $(work_dir_isa)/$$@ > $(work_dir_isa)/$$@.nm; \
-    $$(RISCV_OBJCOPY) -O verilog $(work_dir_isa)/$$@ $(work_dir_isa)/$$@.bin;
+		$(DEFINES) -T$(LDSCRIPT) $$(<) \
+		-o $$@; \
+    $$(RISCV_OBJDUMP) -D $$@ > $$@.objdump; \
+    $$(RISCV_READELF) -a $$@ > $$@.readelf; \
+    $$(RISCV_NM) $$@ > $$@.nm; \
+    $$(RISCV_OBJCOPY) -O verilog $$@ $$@.bin;

